Razor/UI/Config.cs

```diff
@@ -693,6 +693,12 @@ public static Profile CurrentProfile
             }
         }
 
+        public static void AutoSave()
+        {
+            bool autoSaveDisabled = Config.GetAppSetting<bool>("DisableAutoSave");

```

Comment:
Is this feature profile specific, or per profile?

Reply:
I started out using AppSettings but made more sense to me if it was Profile specific, since some profiles are shared and some are not.
So I changed it in commit https://github.com/markdwags/Razor/pull/84/commits/fbbae481c50c7ad6806f2b30cf90bd7aca31b0af